apps/atomy-q/API/app/Http/Controllers/Api/V1/ReportController.php

```
@@ -162,9 +125,9 @@ public function generateSpendByCategorySummary(Request $request): JsonResponse
     public function export(Request $request): JsonResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)